quickstarts/android/03-session-handling.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Auth0 Android Quickstarts - 3. Session Handling
+description: This tutorial will show you how to use Lock10 to mantain a session connected.
+---
+
+## Android - Session Handling Tutorial
+
+This tutorial will show you how to use Lock10 to mantain an active session with Auth0.
+
+The user's Credentials, contains three kind of tokens:
+
+* idToken: Identity Token that proves the identity of the user.
+* accessToken: Access Token used by Auth0 API.
+* refreshToken: Refresh Token that can be used to request new tokens without signing in again.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+  :::
+
+
+### Before Starting
+
+Be sure that you have completed the [Basic Login](01-login.md) Quickstart.
+
+### 1. Save the User's Credential
+
+Your first step is to save, through a secure method, the user's credentials obtained in the login success response. We won't cover how to do that on this tutorial, feel free to save it.
```

Comment:
Last sentence here sounds incomplete... I guess it should be something like "feel free to save it as you like." or something like that.